examples/templateservicebroker/apiserver-config.yaml

```diff
@@ -0,0 +1,4 @@
+kind: TemplateServiceBrokerConfig
+apiVersion: config.templateservicebroker.openshift.io/v1
+templateNamespaces:
+- openshift
```

Comment:
> If we're going to keep this contained within origin, would it make sense to create a role that will ensure this template exists and will apply it (along with all other TSB specific files)?

We should talk about this bit.  I expected that the ensure/apply logic would live in ansible and be consistent across *all* components (like service catalog) that are trying to install server components.  They would provide:
 1. config.yaml files
 2. RBAC template which you process and then reconcile (command to come).
 3. Template which you process, add "normal" parameters, add the config.yaml as a parameter, and apply.

Doing that will allow the component owners to keep their templates, roles, and config.yaml's up to  date in origin.